collector/common/op_counters.go

```diff
@@ -19,14 +19,14 @@ import (
 )
 
 var (
-	opCountersTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	opCountersTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
> These numbers will grow over time

That's counter.

The same thing for one metric below. Please update it in a new PR.